cpp/include/cuspatial/shapefile_reader.hpp

```diff
@@ -23,6 +23,11 @@
 
 namespace cuspatial {
 
+typedef enum winding_order : bool {
```

Comment:
For type safety.

```suggestion
typedef enum class winding_order : bool {
```

Reply:
Can I get you to bend on this one? The enum is linked to via Cython and Cython doesn't like using the `class` identifier here. I don't know if there is a workaround and finding one might take hours.

Reply:
Ok, thanks to @shwina this is resolved.